modules/setting/cache.go

```diff
@@ -58,11 +58,16 @@ func newCacheService() {
 		log.Fatal("Failed to map Cache settings: %v", err)
 	}
 
-	CacheService.Adapter = sec.Key("ADAPTER").In("memory", []string{"memory", "redis", "memcache"})
+	CacheService.Adapter = sec.Key("ADAPTER").In("memory", []string{"memory", "redis", "memcache", "twoqueue"})
 	switch CacheService.Adapter {
 	case "memory":
 	case "redis", "memcache":
 		CacheService.Conn = strings.Trim(sec.Key("HOST").String(), "\" ")
+	case "twoqueue":
+		CacheService.Conn = strings.TrimSpace(sec.Key("HOST").String())
+		if CacheService.Conn == "" {
+			CacheService.Conn = "50000"
```

Comment:
Limit should be a new option so that other adapters have also opportunity to limit the cache size.

Reply:
For Redis and Memcache that limit would usually be expressed in the configuration of the server.

Reply:
I think it's ok to reuse it. - it's well documented